index.js

```diff
@@ -1,38 +1,37 @@
 import { tabbable, focusable, isFocusable, isTabbable } from 'tabbable';
 
-const activeFocusTraps = (function () {
-  const trapQueue = [];
-  return {
-    activateTrap(trap) {
-      if (trapQueue.length > 0) {
-        const activeTrap = trapQueue[trapQueue.length - 1];
-        if (activeTrap !== trap) {
-          activeTrap.pause();
-        }
+const rooTrapStack = [];
```

Comment:
I don't think we need to declare this here at all, and I'd rather not so that there's no tendency to misuse this array in the rest of the code.

Reply:
We must keep it global

Since we can have a lot of instances of focus-trap, it must share the same trapStack

Reply:
Oh, yes, you're right. It shouldn't go into `createFocusTrap()` as I suggested. 🤦‍♂️